library/src/main/java/in/krharsh17/barview/BarGroup.java

```diff
@@ -325,6 +397,13 @@ public boolean willChangeBounds() {
             return true;
         }
     }
+
+    /**
+     * ?
+     *
+     * @param color
+     * @return
+     */
```

Comment:
@Alaska18 Please provide the documentation for this

Reply:
Please do reply @Alaska18

Reply:
This creates a gradient drawable background for the bars - sets a shape, a color and radius for the corners.
@param color is the color in int which is  used as the background color.